### ISSUE TITLE
Multiple rename

### ISSUE THREAD

**Issue by CakesTwix** [equeim/tremotesf2#201]
Hello, I would like to have the function of mass renaming files in a torrent.
![image](https://user-images.githubusercontent.com/57946485/193459673-fd69159d-2f00-4f6d-8289-071a3d3fa310.png)
*here the Rename button is gray*

For example, in Dolphin it is done like this
![image](https://user-images.githubusercontent.com/57946485/193459732-3511c602-e42e-4f3d-be41-5dad55870a06.png)

**Comment by SanyaPilot** [comment]
Yeah, this is really useful feature, especially when downloading media for home-streaming software (like Jellyfin)

**Comment by KiwiNicer** [comment]
I haven't seen this feature anywhere in torrent clients, so it will be a reason to switch to this client

**Comment by gmankab** [comment]
i really need this too
